config.xmi

```diff
@@ -157,6 +157,11 @@
     <title text="Schutzgebiete"/>
     <objectType>ps_4.0:ProtectedSite</objectType>
   </layerConfig>
+  <layerConfig name="PS_ProtectedSite" registryId="http://inspire.ec.europa.eu/layer/PS.ProtectedSite" tags="//@tags.0 //@tags.2" layerName="PS.ProtectedSite" styleConfig="//@styleConfig.106">
```

Comment:
The style config id is the same as for the PS 4.0 layer. I assume it needs to be changed to match the new style added for 5.0.